DigitalLearningSolutions.Data/Models/LearningHubApiClient/BulkResourceReference.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DigitalLearningSolutions.Data.Models.LearningHubApiClient
+{
+    using System.Collections.Generic;
+
+    public class BulkResourceReference

```

Comment:
I'd be inclined to keep this plural - you have lists of resource references

Reply:
This matches the naming of the API BulkResourceReferenceViewModel.